tests/test_aiobaro.py

```diff
@@ -274,52 +274,89 @@ async def test_thumbnail(matrix_client):
     assert result.ok
 
 
-async def test_profile_get(matrix_client):
-    args, kwargs = [], {}
-    result = await matrix_client.profile_get(*args, **kwargs)
+@pytest.mark.asyncio
+async def test_profile_set_displayname(matrix_client):
+    await test_register(matrix_client)
+    await test_login(matrix_client)
+
+    user_id = "@test_user:baro"
+    display_name = "Test USER"
+    result = await matrix_client.profile_set_displayname(user_id, display_name)
     assert result.ok
 
 
+@pytest.mark.asyncio
 async def test_profile_get_displayname(matrix_client):
-    args, kwargs = [], {}
-    result = await matrix_client.profile_get_displayname(*args, **kwargs)
+    await test_profile_set_displayname(matrix_client)
+
+    user_id = "@test_user:baro"
+    result = await matrix_client.profile_get_displayname(user_id)
     assert result.ok
+    assert result.json().get("displayname")
```

Comment:
assert response contains the key but you must check the value

Reply:
Thanks, will be done!